src/test/java/com/jcabi/s3/AwsBucketTest.java

```diff
@@ -106,4 +115,119 @@ public void existsThrowsIOException() throws IOException {
         bucket.exists();
     }
 
+    /**
+     * AwsBucket supports listing large buckets.
+     * @throws Exception if test fails
+     */
+    @Test
+    public void supportsListingLargeBuckets() throws Exception {
+        final String bucketName = "large.bucket";
+        final String prefix = "prefix";
+        final String firstItem = "first";
+        final String secondItem = "second";
```

Comment:
do we have something else that is `first` here? like `firstName` or `firstFile`? no, we don't. than why this suffix? just `first` and `second` will be clear enough